### DISCUSSION TITLE
static get visible() with id and username returns 0

### DISCUSSION THREAD

**Discussion by punity-team** [adonisjs/core#1646] [General]
HI!

I have the code 

```
static get visible() {
    return ['id','username']
  }
```

in my user model.

When I try to create a user, I get this result:
```

{
    "message:": "successfully created",
    "user": {
        "id": 0,
        "username": "test"
    }
}
```

Why I get 0 as ID ?

**Comment by thetutlage** [comment]
Can you please format your codeblocks. Here's a link to the markdown guide https://guides.github.com/features/mastering-markdown/

**Reply by punity-team** [reply]
yeah. can you help?

**Reply by punity-team** [reply]
@thetutlage ...

**Comment by punity-team** [comment]
i've fixed myself, I needed to insert this code into my model:


```
static get incrementing() {
    return false
}
```
